### PR TITLE
HCLIND-47 Fix: ITAM policies are hardcoded for NAM

### PR DESCRIPTION
**Pull request by juanestrahcl**
### Description

Replaced hardcoded host in all ds using f1_app_host as base.

### Issues Resolved

ITAM policies are hardcoded for NAM, more info: [Here](https://flexera.atlassian.net/browse/HCLIND-47).

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: compliance/fnms/missing_active_machines/missing_active_machines.pt [hunk 1]
```diff
@@ -46,7 +46,7 @@ end
 datasource "ds_inventories" do
   request do
     auth $auth_flexeraone
-    host "api.flexera.com"
+    host join(["api.", get(1, split(f1_app_host, "app."))])
```
Comment (degarajesh) [inline compliance/fnms/missing_active_machines/missing_active_machines.pt:49]: Please add the version

**Review by degarajesh — COMMENTED** [review]
LGTM.

**Review by nia-vf1 — APPROVED** [review]
LGTM

**Comment by juanestrahcl** [comment]
Code was tested on North America (app.flexera.com & app.flexeratest.com), EU (app.flexera.eu) and APAC (app.flexera.au), in all regions it's working well.

**Review by degarajesh — APPROVED** [review]

**Pull request merged**